tests/unit/plugins/v1/test_catkin.py

```diff
@@ -1909,7 +1910,7 @@ def test_find_system_dependencies_system_only(self):
 
         self.assertThat(
             catkin._find_system_dependencies(
-                {"foo"}, self.rosdep_mock, self.catkin_mock
+                {"foo"}, self.rosdep_mock, self.catkin_mock, {}
```

Comment:
Note that you're passing a set in for skip keys, but skip keys in reality will be a list. Probably not a big deal, but something to consider and proof that we need some mypy magic in the catkin plugin.

Reply:
Wait no... this is a dict. Why? How? :stuck_out_tongue:

Reply:
should be fixed